src/betterproto/__init__.py

```diff
@@ -551,6 +551,12 @@ def __setattr__(self, attr: str, value: Any) -> None:
 
         super().__setattr__(attr, value)
 
+    def __bool__(self) -> bool:
+        return any(
+            getattr(self, field_name)
```

Comment:
would it make more sense to say
```suggestion
            getattr(self, field_name) is not None
```
?

Reply:
I'm just gonna assume at this point #130 will be merged so I'm updating for that